oam/oamcpp/liboamcpp.cpp

```diff
@@ -8795,11 +8799,6 @@ int Oam::glusterctl(GLUSTER_COMMANDS command, std::string argument1, std::string
 
     string glustercmd = "gluster ";
 
-    if (user != 0)
-    {
-        glustercmd = "sudo " + glustercmd;
```

Comment:
if any of the gluster cmds below do need root perms then they won't work as not-root.